lib/identity_cors.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class IdentityCors
-  FEDERALIST_REGEX = %r{\Ahttps://federalist-[0-9a-f-]+\.pages\.cloud\.gov\z}
+  FEDERALIST_REGEX = %r{\Ahttps://federalist-[0-9a-f-]+\.sites\.pages\.cloud\.gov\z}
```

Comment:
one more thought, are there old preview domains that this worked on? should we make this an optoinal segment?
```suggestion
  FEDERALIST_REGEX = %r{\Ahttps://federalist-[0-9a-f-]+\(.sites)?\.pages\.cloud\.gov\z}
```

Reply:
Just noting: I think there's an error in Zach's suggestion, the escape should be applied to the dot, not the parenthesis.

```suggestion
  FEDERALIST_REGEX = %r{\Ahttps://federalist-[0-9a-f-]+(\.sites)?\.pages\.cloud\.gov\z}
```

Reply:
bah I thought I could sneak in my fix before anybody else noticed!

Reply:
Though to the broader question, I'm not sure if we need to be backwards-compatible here. At least, best I can tell is that these have been the [preview URLs host for at least a year](https://github.com/cloud-gov/pages-core/commit/016b888b486ceba2fa49a335e5e69fe12f6ad597), and I don't know if we care too much about legacy preview branches, if they ever were using another domain?